src/app/pages/CreateConnectorPage/StepConfigurator.tsx

```diff
@@ -60,7 +60,7 @@ const ConnectedCustomConfigurator: FunctionComponent<{
       activeStep={activeStep}
       configuration={configuration}
       connector={connector}
-      isViewMode={false}
+      isViewMode={duplicateMode && false}
```

Comment:
@indraraj is this a logical "and"? Then this would still always be false, won't it?

Reply:
yes and no.
`undefined && false`  is `undefined`